tools/validate_configs/test_configs/htcondor-pool.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: htcondor-pool
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: htcondor-001
+  region: us-central1
+  zone: us-central1-c
+  image_family: htcondor-9x
+  network_name: htcondor-pool
+  subnetwork_name: htcondor-pool-usc1
+
+deployment_groups:
+- group: htcondor-env
+  modules:
+  - source: modules/network/vpc
+    kind: terraform
+    id: network1
+    outputs:
+    - network_name
+  - source: community/modules/scripts/htcondor-install
+    kind: terraform
+    id: htcondor_install
+  - source: community/modules/project/service-enablement
+    kind: terraform
+    id: htcondor_services
+    use:
+    - htcondor_install
+  - source: modules/scripts/startup-script
+    kind: terraform
+    id: htcondor_install_scripts
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - $(htcondor_install.install_htcondor_runner)
+    outputs:
+    - startup_script
+- group: packer
```

Comment:
nit: I prefer to put whitespace between the groups, similar to what is in image-builder.yaml.

Reply:
I do not prefer the degree to which YAML allows for empty lines, but I will match the rest of our examples in style.

Reply:
Done

Reply:
You did go a step farther than I was asking. The request was to put white space between the 3 groups, not every module, which would match how image-builder.yaml is. But this is a nit so feel free to close as you see fit.

Reply:
I decided to interpret the nit as "match the style of files under community/examples" since an example like this might eventually land there. I don't like how wild west YAML can be with respect to whitespace.